symbolic-symcache/src/format.rs

```diff
@@ -298,17 +299,17 @@ pub struct HeaderV2 {
     pub preamble: Preamble,
     /// Debug identifier of the object file.
     pub debug_id: DebugId,
-    /// CPU architecture of the debug file.
+    /// CPU architecture of the debug file. NOTE: debug file or object file?
```

Comment:
This is equivalent, but since we're mentioning "object file" above you can go with that here, too.